R/import_gtfs.R

```diff
@@ -368,6 +369,34 @@ read_files <- function(file,
 
 }
 
+#' Read geojson file
+#'
+#' @param file.geojson geojson file
+#'
+#' @keywords internal
+#' @importFrom jsonlite read_json
+read_geojson <- function(file.geojson) {
+  read_json(file.geojson)
+}
+
+remove_file_ext = function(file) {
+  gsub("\\.txt$", "", gsub("\\.geojson$", "", file))
```

Comment:
```suggestion
  tools::file_path_sans_ext(file)
```

Reply:
Hi @polettif , I think it will have no effect on gtfstools, but we will be making a few tests in the next couple of days and report here.

Reply:
Thanks for the swift response!